ras_party/controllers/account_controller.py

```diff
@@ -275,23 +274,24 @@ def request_password_change(payload, session):
     if not respondent:
         raise RasError("Respondent does not exist.", status=404)
 
-    email_address = respondent.email_address
-    verification_url = PublicWebsite().reset_password_url(email_address)
+    if respondent.status is RespondentStatus.ACTIVE:
```

Comment:
I think we need some debug level logging on this. Just a "Requesting password change" and "Successfully changed password" with party_id potentially